src/tests/ftest/util/ior_utils.py

```diff
@@ -467,6 +486,18 @@ def command(self):
         """
         return self.manager.job
 
+    def update(self, name, value):
+        """Update a IorCommand BasicParameter with a new value.
+
+        Args:
+            name (str): name of the IorCommand BasicParameter to update
+            value (str): value to assign to the IorCommand BasicParameter
+        """
+        param = getattr(self.command, name, None)
+        if param:
+            if isinstance(param, BasicParameter):
+                param.update(value, ".".join([self.command.command, name]))
+
```

Comment:
Suggestion - how about defining `update_params` similar to `ObjectWithParameters.update_params`?